deploy/docker/cp-edge/sync-routes.py

```diff
@@ -811,7 +822,8 @@ def update_svc_url_for_run(run_id):
 for added_route in routes_to_add:
         service_spec = services_list[added_route]
 
-        if service_spec["create_dns_record"] and not service_spec["custom_domain"]:
+        if service_spec["create_dns_record"] and not service_spec["custom_domain"] \
+                and default_edge_region == edge_region:
```

Comment:
I believe we should look at this `if` closely
so if `service_spec["create_dns_record"] and not service_spec["custom_domain"]` is true but `default_edge_region == edge_region` is false? we will go to `else` branch and create simple `create_service_location` and also update a link for this region 
but as I understand we want create link only for default edge and do nothing for the others